src/openrct2-ui/windows/TileInspector.cpp

```diff
@@ -2094,26 +2094,6 @@ static void window_tile_inspector_paint(rct_window* w, rct_drawpixelinfo* dpi)
                 int16_t wallType = tileElement->AsWall()->GetEntryIndex();
                 DrawTextBasic(dpi, screenCoords, STR_TILE_INSPECTOR_WALL_TYPE, &wallType, { COLOUR_WHITE });
 
-                // Banner info
-                const auto wallEntry = tileElement->AsWall()->GetEntry();
-                if (wallEntry != nullptr && wallEntry->wall.flags & WALL_SCENERY_IS_BANNER)
```

Comment:
I meant remove this specific if check. Walls can have banners but the correct way to check if they exist or not is using `auto banner = tileElement->AsWall()->GetBanner();` and checking if the result is non null.